### DISCUSSION TITLE
android HA application does not always connect through cloudflared

### DISCUSSION THREAD

**Discussion by RichardPar** [homeassistant-apps/app-cloudflared#394] [Q&A]
I use the Android app for GeoFence automations - Since using Cloudflared, I have to open the HA app and 'wait' for it to login before the position is updated. I get the error 'Unable to connect' but within 3 seconds, it clears the error and connects. 

Connecting without cloudflared (using a port forward) - the Geofence functionality works.

Is this a known issue? .. or am I barking up the wrong tree?

Regards,
Richard

**Comment by brenner-tobias** [comment]
Are you using an additional access policy in an application setup in the Cloudflared Zero Trust Dashboard?

**Comment by RichardPar** [comment]
Bog standard... https redirect always - nothing else
